kt/godot-library/src/main/kotlin/godot/tests.kt

```diff
@@ -6,9 +6,9 @@ import godot.util.VoidPtr
 import godot.util.camelToSnakeCase
 import kotlin.reflect.KCallable
 
-open class Object(isRef: Boolean = false) : KtObject(isRef) {
+open class Object internal constructor(isRef: Boolean = false, isSingleton: Boolean = false) : KtObject(isRef, isSingleton) {
```

Comment:
`isSingleton` (and `isRef`) is a property of the class not the instance, so it is quite weird to have it as instance fields. Is there a way we can use the types for this checks?

Reply:
No way to use the type and get performances :/
I think I'll wait for merge of other PR and implement the check with internal method with unifying of godot-library and godot-runtime modules